aws-observability/scripts/AWSOAutoSetupScript/README.md

```diff
@@ -1,40 +1,45 @@
 ## Pre-Requisite
 [AWS CLI](https://aws.amazon.com/cli/)
 
-# For installing on Mac and Linux, here are the instructions
 # AWS Observability Deployment using Script
+## For installing on Mac and Linux
+### Command
 The `DeployAWSOPosix.sh` enables you to deploy the AWS Observability CloudFromation template using AWS CLI commands.
-
-
-
-## Parameters
-Script takes two inputs
-1. SUMO ACCESS ID - Provide the Sumo Access Id from your respective Sumo Logic Account where you want to install AWS Observability Solution
-2. SUMO ACCESS KEY - Provide the Sumo Access Key from your respective Sumo Logic Account where you want to install AWS Observability Solution
-
-AWS_PROFILE can be set as environment variable from the Command Line before executing the sh file. If it is not set, default aws profile will be picked from the local machine.
-
-## Command
-Below is an example to run the sh file with respective parameters
  
-`./DeployAWSOPosix.sh <SUMO_ACCESS_ID> <SUMO_ACCESS_KEY>`
+```
+wget "https://raw.githubusercontent.com/SumoLogic/sumologic-solution-templates/master/aws-observability/scripts/AWSOAutoSetupScript/DeployAWSOPosix.sh"
 
+chmod +x DeployAWSOPosix.sh
 
+./DeployAWSOPosix.sh -i <SUMO_ACCESS_ID> -k <SUMO_ACCESS_KEY> -p <AWS_PROFILE> -r <AWS_REGION>
+```
```

Comment:
The above commands will perform following actions:
Downloading master template using wget command.
Using chmod command to grant execute permission to shell script.
Execute shell script for deploying the solution.